client/my-sites/stats/stats-followers/index.jsx

```diff
@@ -80,11 +77,13 @@ class StatModuleFollowers extends Component {
 		const summaryPageSlug = siteSlug || '';
 		// email-followers is no longer available, so fallback to the new subscribers URL.
 		// Old, non-functional path: '/people/email-followers/' + summaryPageSlug.
-		// If the site is Atomic or Jetpack self-hosted, it links to Jetpack cloud.
-		// For all other cases, turn to the WP.com URL since Odyssey Stats doesn't have the Followers page.
+		// Subscribers have been moved to Jetpack Cloud.
+		// Once we release Simple Site Management in Jetpack Cloud, we will use the Jetpack Cloud link.
 		const jetpackCloudLink = `https://cloud.jetpack.com/subscribers/${ summaryPageSlug }`;
 		const wpcomLink = `https://wordpress.com/people/subscribers/${ summaryPageSlug }`;
-		const summaryPageLink = isAtomic || isJetpack ? jetpackCloudLink : wpcomLink;
+		const summaryPageLink = isEnabled( 'jetpack/manage-simple-sites' )
```

Comment:
I didn't get a chance to test this PR functionally today. Just observing the code, should this continue to include Jetpack and Atomic like: `isAtomic || isJetpack || isEnabled( 'jetpack/manage-simple-sites' )`?

Reply:
Making a note for myself: We only want to go to wp.com/people/subscribers for Simple sites.

Yeah I agree keeping the flags you mentioned is safer.